pallets/gear/src/runtime_api.rs

```diff
@@ -308,7 +308,8 @@ where
 
     fn code_with_memory(program_id: ProgramId) -> Result<CodeWithMemoryData, String> {
         let program = ProgramStorageOf::<T>::get_program(program_id)
```

Comment:
```suggestion
        let (program, _bn) = ProgramStorageOf::<T>::get_program(program_id)
```
Same readability stuff

Reply:
done